### ISSUE TITLE
Have you done some experiments on DeeplabV3plus? How's the performance?

### ISSUE THREAD

**Issue by liruilong940607** [kazuto1011/deeplab-pytorch#9]

**Comment by kazuto1011** [comment]
I haven't tried appropriate training yet but it showed 10% lower results with the same setting with v2. I'm not sure but it seemed to be similar to the v2 result with an un-frozen batch normalization. I plan to implement the v3+ training but I don't have time now, sorry.

**Comment by Engineering-Course** [comment]
@kazuto1011  
This repository contains DeepLab v3/v3+ model definition.
Is it right that I can train with v3+ once I implement the training scripts?

**Comment by kazuto1011** [comment]
I think you can train the model but I have just read the papers. I carefully implemented those models and tried training them with v2 scheme, but it was not to replicate results. Please let me know if it wouldn't work.
